### DISCUSSION TITLE
Unknown at rule @theme in style.css

### DISCUSSION THREAD

**Discussion by antonijo01** [tailwindlabs/tailwindcss#13205] [Help]
**What version of Tailwind CSS are you using?**

v4-alpha

**What build tool (or framework if it abstracts the build tool) are you using?**

VSCode

**What version of Node.js are you using?**

Node v21.0.0.

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Reproduction URL**

localhost

**Describe your issue**

After installing via CLI, I tried to add @theme to style.css and got an error.

![image](https://github.com/tailwindlabs/tailwindcss/assets/107608985/f9cbc3d4-b14d-4a6e-8f85-3da5163d9de0)

**Comment by wongjn** [comment]
It's a warning from VSCode's in-built CSS parser/validator in that it does not recognize the `@theme` at rule, since it is indeed not a CSS feature but specific to Tailwind. Nothing to worry about and it should not directly impact using Tailwind.

**Comment by depsimon** [comment]
You can fix this by changing the Language Mode to Tailwind CSS if you use the Tailwind CSS IntelliSense VS Code extension.

**Reply by candidia** [reply]
When you do that, you can no longer format the file: "There is no formatter for 'tailwindcss' files installed."

**Reply by LestevMisha** [reply]
Hey, I was dealing with the same issue, there was no extensions for it out there. So I made one for myself, if you still need it, here https://marketplace.visualstudio.com/items?itemName=MishaLestev.tailwind-css-language-mode-formatter (Tailwind CSS Language Mode Formatter). [On Github](https://github.com/LestevMisha/Tailwind-CSS-Language-Mode-Formatter)

**Comment by Lord-Gown** [comment]
Just install the postcss language support extension for vscode since tailwind uses that also

**Comment by sydneyvaughn75-ops** [comment]
Tested and it good thank you
